java/src/test/java/com/google/openlocationcode/EncodingTest.java

```diff
@@ -57,30 +55,23 @@ public void setUp() throws Exception {
     }
   }
 
-  // Gets the test file, factoring in whether it's being built from Maven or Bazel.
-  private File getTestFile() {
-    String testPath;
-    String bazelRootPath = System.getenv("JAVA_RUNFILES");
-    if (bazelRootPath == null) {
-      File userDir = new File(System.getProperty("user.dir"));
-      testPath = userDir.getParent() + "/test_data";
-    } else {
-      testPath = bazelRootPath + "/openlocationcode/test_data";
-    }
-    return new File(testPath, "encoding.csv");
-  }
-
   @Test
   public void testEncodeFromLatLong() {
     for (TestData testData : testDataList) {
+      double latitude = testData.latitude;
+      double longitude = testData.longitude;
+      int length = testData.length;
+      String encode = OpenLocationCode.encode(latitude, longitude, length);
       Assert.assertEquals(
-          String.format(
-              "Latitude %f, longitude %f and length %d were wrongly encoded.",
-              testData.latitude,
-              testData.longitude,
-              testData.length),
+          "Latitude :"
+              + latitude
+              + ", longitude:"
+              + longitude
+              + " and length:"
+              + length
+              + " were wrongly encoded.",
```

Comment:
Why the change? This looks less readable to me.

Reply:
It's cause Used Google Java code formatting(1.7). 
Added spotless that will apply on java files and formats the code.

Reply:
https://github.com/google/open-location-code/pull/325/files#r282155179

Reply:
Spotless is  to Java, like pylint to python.

Reply:
I agree with @zongweil  - that looks horrible. If you need a formatter please use https://github.com/google/google-java-format (which is what the original code was based on).

Reply:
@drinckes  i have used the same. 
Spotless plugin with https://github.com/google/google-java-format 1.7

Reply:
See this plugin:  https://github.com/google/open-location-code/blob/5dfd58e443f6651f0ff61db991dc8a1fd3ad3b18/java/pom.xml#L90

this creates a plugin profile in IntelliJ, with the help of that you can easily apply the formatting. that is using the same google java formatter 1.7

Reply:
@drinckes @zongweil, I do realize my mistake and corrected. thanks for the review and I do apologize for my mistake.

Reply:
I have updated this, hope it's fine.